internal/cli/atlas/atlas.go

```diff
@@ -49,8 +48,6 @@ func Builder() *cobra.Command {
 		Use:   Use,
 		Short: "Atlas operations.",
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-			config.SetService(config.CloudService)
```

Comment:
can we condition this? if not set set it to cloud?

Reply:
sure